iceoryx_hoofs/include/iceoryx_hoofs/log/platform_building_blocks/logcommon.hpp

```diff
@@ -36,17 +36,21 @@ enum class LogLevel : uint8_t
 };
 
 /// @brief converts LogLevel into a string literal
+/// @param[in] value is the actual LogLevel to convert
```

Comment:
I would remove `is` in here and all the other occurrences of `@param[in] value` in this file.

Reply:
So `value the actual LogLevel to convert`? This reads somehow weird. How about `value of the actual LogLevel to convert`?